src/Workspaces/Core/Portable/Shared/Extensions/ContextQuery/AbstractSyntaxContext.cs

```diff
@@ -87,6 +93,12 @@ private ISet<INamedTypeSymbol> ComputeOuterTypes(CancellationToken cancellationT
             return SpecializedCollections.EmptySet<INamedTypeSymbol>();
         }
 
+        protected static IEnumerable<ITypeSymbol> ComputeInferredTypes(Workspace workspace, SemanticModel semanticModel, int position, CancellationToken cancellationToken)
```

Comment:
It seems super wonky this is here.  Why can't this be computed in the AbstractSyntaxContext constructor?